core/civilization/person/default.py

```diff
@@ -46,6 +51,7 @@ def respond(self, sender: Person, request: str, params: TalkParams) -> str:
             result, is_finish = self.execute(plans[0], sender=sender)
 
             if is_finish:
+                self.mouth.talk(sender.ear.port, result, "")
```

Comment:
mouth talks to ear. not ear's port. agree?
```suggestion
                self.mouth.talk(sender.ear, result, "")
```

Reply:
ah yes yes